prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.ts

```diff
@@ -196,8 +197,15 @@ export class OverviewComponent extends BaseEnrolmentPage implements OnInit {
    * enrolment for checking validation instead of form state.
    */
   private getEnrolmentErrors(enrolment: Enrolment): ValidationErrors {
+    const isDeviceProvider = this.enrolmentService.enrolment.careSettings.some((careSetting) =>
+      careSetting.careSettingCode === CareSettingEnum.DEVICE_PROVIDER);
+    const isDeviceProviderIdentifier = this.enrolmentService.enrolment.deviceProviderIdentifier;
```

Comment:
This would be `const hasDeviceProviderIdentifier`

Reply:
I swear I renamed that from previous comment lol